tests/app.test.js

```diff
@@ -460,4 +459,45 @@ describe('InVEST subprocess testing', () => {
     expect(cardText).toBeInTheDocument();
     unmount();
   })
+
+  test('user terminates process - expect log display', async () => {
+    const spy = jest.spyOn(InvestJob.prototype, 'terminateInvestProcess')
+      .mockImplementation(() => {
+        console.log('click handler')
+        mockInvestProc.emit('exit', null)
+      })
+
+    const { findByText, findByLabelText, unmount, ...utils } = render(
+      <App
+        jobDatabase={fileRegistry.JOBS_DATABASE}
+        investExe='foo'/>);
+
+    const carbon = await findByText('Carbon');
+    fireEvent.click(carbon);
+    const workspaceInput = await findByLabelText(
+      RegExp(`${spec.args.workspace_dir.name}`))
+    fireEvent.change(workspaceInput, { target: { value: fakeWorkspace } })
+    
+    const execute = await findByText('Execute');
+    fireEvent.click(execute);
+    
+    // stdout listener is how the app knows the process started
+    mockInvestProc.stdout.push('hello from stdout');
+    const logTab = await findByText('Log');
+    expect(logTab.classList.contains('active')).toBeTruthy();
+    // some text from the logfile should be rendered:
+    expect(await findByText(dummyTextToLog, { exact: false }))
+      .toBeInTheDocument();
+    const cancelButton = await findByText('Cancel Run');
+    fireEvent.click(cancelButton);
+    expect(await findByText('Run Canceled')).toBeInTheDocument();
```

Comment:
So on Windows this will error because we can't properly `kill` a process?

Reply:
Yes. Tehcnically it fails because that "Run Canceled" message only shows when the subprocess has time to listen and run code on an `exit` event. Right now on Windows we're resorting to a force kill and there's no such time.

Reply:
@phargogh Maybe I mistated that. The callback is executed by the parent. Node (the parent) has a reference to the child process (`this.investRun`) that is listening for events from the child. So maybe the problem here is that the child never has the chance to emit the event?

Reply:
So now that I'm actually testing this on Windows, my last two statements are both wrong. The exit code on Windows after the force-kill is `1`.